dbsync-central/logback-spring.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+	<springProperty name="eip.base.dir" source="eip.home" defaultValue="/home/eip" />
```

Comment:
Is this line the reason you are adding this file because it comes with the jar file already

Reply:
I'm changing the location of logs. But I thing I can let them where they are